HRTool/src/main/java/app/controllers/ImageUploadRestController.java

```diff
@@ -14,55 +12,26 @@
 import org.springframework.web.bind.annotation.ResponseBody;
 import org.springframework.web.multipart.MultipartFile;
 
-
-
 @Controller
-
 public class ImageUploadRestController {
```

Comment:
This controller shouldn't exist. Methods in this controller should belong to a resource